cardano-node/src/Cardano/Node/Protocol/Shelley.hs

```diff
@@ -213,7 +216,7 @@ renderShelleyProtocolInstantiationError pie =
     GenesisDecodeError fp err ->
         "There was an error parsing the genesis file: "
      <> toS fp <> " Error: " <> T.pack (show err)
-
+    GenesisValidationFailure vErrs -> T.unlines $ map describeValidationErr vErrs
```

Comment:
All of the cases here are separated by a newline. I think we should continue doing that for consistency:

```suggestion
    GenesisValidationFailure vErrs -> T.unlines $ map describeValidationErr vErrs
    
```